apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+$red: #ff0000;
+
+.nav {
+	background-image: url("../../assets/images/lined_paper.svg");
```

Comment:
Chore: asset path would be more maintainable and readable as absolute instead of relative.